app/src/main/java/com/example/munchbox/ui/components/OrderSummaries.kt

```diff
@@ -19,10 +19,14 @@ import com.example.munchbox.controller.DayOfWeek
 import com.example.munchbox.controller.Meal
 import com.example.munchbox.data.DataSource
 import com.example.munchbox.data.OrderUiState
+import com.example.munchbox.data.StorageServices
 import com.example.munchbox.ui.OrderViewModel
+import com.google.firebase.firestore.FirebaseFirestore
 
 @Composable
 fun OrderSummaries(order: OrderUiState, modifier: Modifier = Modifier){
+    val orderService = StorageServices(FirebaseFirestore.getInstance()).orderService()
```

Comment:
here too @Seanlebon

Reply:
We can probably pass it down once we set up everything else to be able to take in the service (aka once we restructure the code to user the DB)